Gordon360/Gordon360.csproj

```diff
@@ -25,7 +25,8 @@
     <UseGlobalApplicationHostFile />
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
-    <Use64BitIISExpress>false</Use64BitIISExpress>
+    <Use64BitIISExpress>
+    </Use64BitIISExpress>
```

Comment:
Do you know what this is or why it changed? I've never seen it change before and I'm worried it might affect whether/how the code runs on the API server. But it might also be something that's set at build time depending on the system that is being built for, so I'm not sure whether it matters or not.